src/LtiServiceConnector.php

```diff
@@ -111,7 +91,6 @@ public function makeRequest(IServiceRequest $request)
 
         if ($this->debuggingMode) {
             $this->logRequest(
-                static::AUTH_REQUEST,
```

Comment:
Whoops. We were logging everything like it was an auth request instead of using the specified request type.